ember-container-query/src/components/container-query.ts

```diff
@@ -26,7 +26,7 @@ interface ContainerQuerySignature<T extends IndexSignatureParameter> {
       },
     ];
   };
-  Element: HTMLElement;
+  Element: Element;
```

Comment:
I changed the type from `HTMLElement` to `Element`, to match the signature from `ember-element-helper`.